core/models/document.py

```diff
@@ -30,6 +30,7 @@ class Document(CoreModel):
                    ("consent_form", "Consent Form"),
                    ("subject_informationsheet", "Subject InformationSheet"),
                    ("project_proposal", "Project Proposal"),
+                   ('dpia', 'DPIA'),
```

Comment:
Dear @nene can we replace the short hand  i.e. "DPIA" with the expanded description i.e.  "Data Protection Impact Asssessment"